sopel_waifu/__init__.py

```diff
@@ -112,6 +158,72 @@ def waifu(bot, trigger):
         msg = '{target}, your waifu is {waifu}'
 
     bot.say(msg.format(target=target, waifu=choice))
+    util.cache_waifu(bot, choice, target, trigger.sender)
+
+
+@plugin.command('lastwaifu')
+@plugin.require_chanmsg
+@plugin.output_prefix(OUTPUT_PREFIX)
+@plugin.example('.lastwaifu Peorth', user_help=True)
+@plugin.example('.lastwaifu', user_help=True)
+def last_waifu(bot, trigger):
+    """Get a reminder of someone's last waifu, without picking a new one.
+
+    This is scoped to the current channel.
+    """
+    target = trigger.group(3) or trigger.nick
+
+    cached = util.get_last_waifu(bot, target, trigger.sender)
+    if cached is None:
+        bot.say("{} hasn't gotten a waifu recently.".format(target))
```

Comment:
I didn't use the suggestion _here_ because this message also appears if the user hasn't run `.waifu` since the last bot restart. But I did like this idea and used it to change the after-duel messaging.